css/modal.css

```diff
@@ -0,0 +1,95 @@
+.hero-container {
+  margin-top: 4rem;
+  background-color: var(--light-bg-2);
+}
+
+.modal-container {
+  display: grid;
+  grid-template-rows: 12.4rem 8rem 2rem 5.5rem 3rem;
+}
+.hero-image {
+  width: 300px;
+  height: 350px;
+}
```

Comment:
use rem here

Reply:
thanks for pointing it out.. i have rectified this at other places too